ddocs/medic/views/reports_by_form_and_parent/map.js

```diff
@@ -3,6 +3,6 @@ function(doc) {
       doc.form &&
       doc.contact &&
       doc.contact.parent) {
-    emit([doc.form, doc.contact.parent._id]);
+    emit([doc.form, doc.contact.parent._id], doc.reported_date);
```

Comment:
This view is only used in one place: `shared-libs/transitions/src/lib/utils.js` -> `getReportsWithSameParentAndForm`, which only gets called by `conditional_alerts` transition.